src/javascript/app/pages/cashier/account_transfer.js

```diff
@@ -61,41 +62,40 @@ const AccountTransfer = (() => {
             label.id = 'transfer_to';
 
             el_transfer_to.parentNode.replaceChild(label, el_transfer_to);
-            el_transfer_to = document.getElementById('transfer_to');
+            el_transfer_to = getElementById('transfer_to');
         }
 
         showForm();
 
         if (Client.hasCurrencyType('crypto') && Client.hasCurrencyType('fiat')) {
-            document.getElementById('transfer_fee').setVisibility(1);
+            getElementById('transfer_fee').setVisibility(1);
         }
     };
 
     const hasError = (response) => {
         const error = response.error;
         if (error) {
-            const el_error = document.getElementById('error_message').getElementsByTagName('p')[0];
+            const el_error = getElementById('error_message').getElementsByTagName('p')[0];
             elementTextContent(el_error, error.message);
-            el_error.parentNode.setVisibility(1);
+            if (el_error.parentNpde) {
```

Comment:
minor typo here: `el_error.parentNode` :)

Reply:
thank you!